src/components/molecules/floating-button/demo.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import React from 'react';
+
+import { colors } from '../../../themes';
+import Icon from '../../atoms/icon';
```

Comment:
Good catch, it's a leftover. I originally didn't include Icon in FloatingButton.